doc/use_apps/verify_tx.rst

```diff
@@ -45,4 +45,98 @@ Note that transaction IDs are uniquely assigned by the service - once a request
 
 If the network is unable to reach consensus, it will trigger a leadership election which increments the view. In this case the user's next request may be given a version ``3.16``, followed by ``3.17``, then ``3.18``. The sequence number is reused, but in a different view; the service knows that ``2.18`` can never be assigned, so it can report this as an invalid ID. Read-only transactions are an exception - they do not get a unique transaction ID but instead return the ID of the last write transaction whose state they may have read.
 
-.. note:: To verify a transaction offline, see :ref:`receipts <audit/receipts:Receipts>`.
+Write Receipts
+--------------
+
+Once a transaction has been committed, it is possible to get a cryptographic receipt over the entry produced in the ledger. That receipt can be verified offline.
+
+To obtain a receipt, a user needs to call a :http:GET:`/node/receipt` for a particular transaction ID. Because fetching the information necessary to produce a receipt likely involves a round trip to the ledger, the endpoint is implemented as a historical query.
```

Comment:
```suggestion
To obtain a receipt, a user needs to call a :http:GET:`/app/receipt` for a particular transaction ID. Because fetching the information necessary to produce a receipt likely involves a round trip to the ledger, the endpoint is implemented as a historical query.
```

I know the endpoint is shared by all frontends, but I'm assuming that users are the primary target for this document?